### PR TITLE
Feat/add SubWallet wallet

### PR DESCRIPTION
**Pull request by S2kael**
### Description

**Add SubWallet wallet into web3-onboard.**

SubWallet is the comprehensive non-custodial wallet solution for Polkadot, Substrate & Ethereum ecosystems. With the mission of bringing users closer to Web3, we envision a Web3 multiverse gateway through which users can enjoy multichain services with utmost ease and absolute security. Connecting and using blockchain-based applications is smoother than ever with SubWallet Browser Extension & SubWallet Mobile App.

The provider is available on window.SubWallet.

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [x] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [x] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

### CODE REVIEW

**Comment by taylorjdawson** [comment]
@S2kael Thanks for this addition! Going to take a look at this PR and we'll get it merged  🙌

**Review by taylorjdawson — APPROVED** [review]

File: packages/demo/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "@web3-onboard/gas": "^2.1.7",
     "@web3-onboard/gnosis": "^2.1.9",
     "@web3-onboard/infinity-wallet": "^2.0.3",
-    "@web3-onboard/injected-wallets": "^2.10.5-alpha.1",
+    "@web3-onboard/injected-wallets": "^2.10.6-alpha.1",
```
Comment (taylorjdawson) [inline packages/demo/package.json:40]:
```suggestion
    "@web3-onboard/injected-wallets": "^2.10.5-alpha.2",
```

File: packages/injected/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.10.5-alpha.1",
+  "version": "2.10.6-alpha.1",
```
Comment (taylorjdawson) [inline packages/injected/package.json:3]:
```suggestion
  "version": "2.10.5-alpha.2",
```

**Review by taylorjdawson — CHANGES_REQUESTED** [review]
We just want to bump the alpha version, but besides that it looks good 👍

**Comment by S2kael** [comment]
> We just want to bump the alpha version, but besides that it looks good 👍
Thank you, i just think my pull request can be merge at the next version so i don't change the alpha version. But I updated my version

**Review by taylorjdawson — APPROVED** [review]

**Pull request merged**